config_test.go

```diff
@@ -458,6 +458,29 @@ func TestNutsGlobalConfig_InjectIntoEngine(t *testing.T) {
 		}
 	})
 
+	t.Run("slice param is injected into engine without ConfigKey", func(t *testing.T) {
```

Comment:
Small remark: I'd add a test where a slice isn't a string slice, if that's a possible input coming from YAML.